src/index.ts

```diff
@@ -135,10 +199,10 @@ export class PetraWallet implements AdapterPlugin {
   async onNetworkChange(callback: any): Promise<void> {
     try {
       const handleNetworkChange = async ({
-        name,
-        chainId,
-        url,
-      }: NetworkInfo): Promise<void> => {
+                                           name,
+                                           chainId,
+                                           url,
+                                         }: NetworkInfo): Promise<void> => {
```

Comment:
thats a weird format lol

Reply:
agreed 😆 
I removed the whole thing in #15, if it's cool we can leave it here and hopefully we merge both PRs together?